### ISSUE TITLE
[Mobile iOS] People Picker Navigation

### ISSUE THREAD

**Issue by somoo88** [microsoft/fluentui#19223]
### Environment Information

- **Package version(s)**: "@fluentui/react": "^7.129.0"
- **Browser and OS versions**: Device: iPhone XR, iOS version: 14.0

### Describe the issue:

When using "next field" with VoiceOver controls on iOS, the suggestion list is not responsive. There are some issues being raised due to this flow, and we would like to know whether this is by design.

### Please provide a reproduction of the issue in a codepen:

Video link of the issue (taken from the Fluent UI site, with example people list - no personal information here):
https://user-images.githubusercontent.com/76500054/127921823-e782d49b-fe21-4838-91ce-fcb72108dac2.mp4

#### Actual behavior:

When indicating "next" button using iOS VoiceOver on population of suggestion list, the navigation does not work.

#### Expected behavior:

When indicating "next" button using iOS VoiceOver on population of suggestion list, the navigation through suggestion should work.

**Comment by gouttierre** [comment]
@somoo88 - Thank you for filing this issue with us. We might require further input but our team will let you know once they have managed to review the issue. 

@smhigley - This was detected on iOS Mobile when viewing the site and there is a recording of the issue's repro - would you be able to confirm if the information is enough to follow up on? Or would the team require further input (actual code) into the problem. Thank you

**Comment by smhigley** [comment]
VoiceOver seems like a red herring, since the previous/next field buttons don't work consistently with it off, either.

I tested out the prev/next form field buttons a bit on iOS, and they seem fairly tied to visual appearance, rather than DOM. For example, it skipped both the partially out-of-view third input in this fiddle, as well as the intentionally obscured fifth input: https://jsfiddle.net/r1nkt0h9/2/.

My guess would be that either the picker dropdown is overlaying the next field, or some other styles on the page are interfering with the way iOS handles finding the next form field. This doesn't seem to be an issue with the picker component itself though, since it works fine in this reduced test case: https://codepen.io/smhigley/pen/e942a157be7969f07e882d0e201ae544.
